content/200-orm/050-overview/500-databases/200-database-drivers.mdx

```diff
@@ -35,10 +35,10 @@ You can connect to your database using a Node.js-based driver from Prisma Client
 
 Database providers, such as Neon and PlanetScale, allow you to connect to your database using other protocols besides TCP, such as HTTP and WebSockets. These database drivers are optimized for connecting to your database in serverless and edge environments.
 
-Prisma maintains the following serverless driver adapters:
+Prisma ORM maintains the following serverless driver adapters:
 
-- [Neon](/orm/overview/databases/neon#how-to-use-neons-serverless-driver-with-prisma-preview)
-- [PlanetScale](/orm/overview/databases/planetscale#how-to-use-the-planetscale-serverless-driver-with-prisma-preview)
+- [Neon](/orm/overview/databases/neon#how-to-use-neons-serverless-driver-with-prisma-ormpreview)
```

Comment:
This seems to be missing a dash between "orm" and "preview", probably wrong in the actual page. (Let's see if I see it when I get to review that page)

Reply:
yea this looks to be a typo. although the page loads https://docs-git-prisma-orm-language-change-prisma.vercel.app/orm/overview/databases/neon#how-to-use-neons-serverless-driver-with-prisma-ormpreview 

interesting. 

@grpatel how should we fix this?

Reply:
I found the source when proceeding with the review: https://github.com/prisma/docs/pull/5631#pullrequestreview-1880023375 When this is fixed, it will also need the dash here.

Reply:
Ya I changed the links to match the headings. The heading didn't have a space before but will change it now. Thanks for fixing